.travis.yml

```diff
@@ -29,7 +29,10 @@ before_install:
   - conda info -a
 
 install:
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION $CONDA_PACKAGES
+  - conda install anaconda=2.4
```

Comment:
I don't like installing all of anaconda on travis. It downloads so many packages but it doesn't use most of them. This is a big waste of time and resources for the travis server. The results really should not depend on the anaconda version, because you are specifying the exact environment with the next line.